src/agent/README.md

```diff
@@ -43,12 +43,16 @@ The `rust-agent` depends on [`grpc-rs`](https://github.com/pingcap/grpc-rs) by P
 - https://github.com/alipay/grpc-rs/tree/rust_agent
 
 ### Build from Source
-The rust-agent need to be built with rust nightly, and static linked with musl.
+The rust-agent need to be built with static linked with musl:
 ```bash
-rustup target add x86_64-unknown-linux-musl
-git submodule update --init --recursive  
-sudo ln -s /usr/bin/g++ /bin/musl-g++  
-cargo build --target x86_64-unknown-linux-musl --release
+$ rustup target add x86_64-unknown-linux-musl
+$ sudo ln -s /usr/bin/g++ /bin/musl-g++  
+```
+Download the kata-containers source and build the agent:
+```bash
+$ go get -d -u github.com/kata-containers/kata-containers
```

Comment:
This is slightly awkward. Although this is the simplest method to get the code, it assumes you have golang installed. But rust devs may not have. Also, we're trying to move away from using [`go get`](https://github.com/kata-containers/kata-containers/issues/823) for 2.0, so could you change this to:

```suggestion
$ GOPATH="${GOPATH:-$HOME/go}"
$ dir="$GOPATH/src/github.com/kata-containers"
$ mkdir -p "$dir" && cd "$dir"
$ git clone --depth 1 https://github.com/kata-containers/kata-containers && cd kata-containers
```

/cc @bergwolf